OPi5_CIDATA/user-data

```diff
@@ -0,0 +1,118 @@
+#cloud-config
+
+# This is the user-data configuration file for cloud-init. By default this sets
+# up an initial user called "ubuntu" with password "ubuntu", which must be
+# changed at first login. However, many additional actions can be initiated on
+# first boot from this file. The cloud-init documentation has more details:
+#
+# https://cloudinit.readthedocs.io/
+#
+# Please note that the YAML format employed by this file is sensitive to
+# differences in whitespace; if you are editing this file in an editor (like
+# Notepad) which uses literal tabs, take care to only use spaces for
+# indentation. See the following link for more details:
+#
+# https://en.wikipedia.org/wiki/YAML
+#
+# Some additional examples are provided in comments below the default
+# configuration.
+
+# The (default) ubuntu user must be in the video group for hardware acceleration
+users:
+  - default
+  - name: ubuntu
+    groups: video
+
+# On first boot, set the (default) ubuntu user's password to "ubuntu" and
+# expire user passwords
+chpasswd:
+  expire: false
+  users:
+  - name: ubuntu
+    password: photon
```

Comment:
we'll need to update our downstream documentation with the new login info

Reply:
pi/raspberry still works. This is a small change to the default ubuntu/ubuntu username so I didn't have to change the password every time I ssh'd in to a new image. I can change it back to ubuntu/ubuntu.